.github/ISSUE_TEMPLATE/V&V_request.md

```diff
@@ -21,22 +21,24 @@ Note: not applicable items will be struck through
 **Additional Comments?**
 
 
-
 ***V&V Checklist:***
 
-@biosensics/engineering : 
+**@biosensics/engineering :** 
 - [ ] Team lead determine if full or partial V&V is needed
+
+Server:
 - [ ] Request Server ECO number from [Quality](mailto:quality@biosensics.com)
+- [ ] Make sure github server secrets are set
```

Comment:
`GitHub`. What are the secrets that need to be set? If we can't list them on, can we list the core/required ones?

Reply:
These are the required ones are
```
ROBOT_PERSONAL_ACCESS_TOKEN # Github bot token
DOCKER_USERNAME # Docker Username
DOCKER_PASSWORD # Docker Password
APTIBLE_USERNAME # Aptible bot username
APTIBLE_PASSWORD # Aptible bot password
SSH_PRIVATE_KEY # SSH private key
KNOWN_HOSTS # Known hosts
APTIBLE_TEST_SERVER_URL # The TEST server's git remote url on aptible (ends in .git)
APTIBLE_TEST_APP_NAME # The TEST server's application name on aptible
APTIBLE_PROD_SERVER_URL # The PRODUCTION server's git remote url on aptible (ends in .git)
APTIBLE_PROD_APP_NAME # The PRODUCTION server's application name on aptible
```

Reply:
we want all of those secrets included in the list? @barchard

Reply:
Some of these should already be set as Org secrets. @andrewalc @MarcDerhammer confirm which should be repo-level secrets and Org-level secrets as not all of these are actionable.

Reply:
i added those 7 things to the checklist